### ISSUE TITLE
VanillaLoot/Simple_Dungeon.nbt not found WARN

### ISSUE THREAD

**Issue by andris155** [BiomeBundle/BiomeBundle#91]
```
[01:22:34] [Server thread/INFO]: [Multiverse-Core] Enabling Multiverse-Core v2.5.0-b727
[01:22:35] [Server thread/WARN]: [OTG] NBT file plugins/OpenTerrainGenerator/worlds/world2/WorldObjects/Structures/BranchingStructures/OvergrownDungeon/../../nbt/VanillaLoot/Simple_Dungeon.nbt not found
[01:22:44] [Server thread/INFO]: [OTG] 2231 world custom objects loaded.
[01:22:44] [Server thread/INFO]: [OTG] 452 biomes Loaded
```

**Comment by LordSmellyPants** [comment]
Thanks for the post, will look into!

**Comment by yuriy-chumak** [comment]
Just update in `WorldObjects/Structures/BranchingStructures/OvergrownDungeon/*` files with
`sed -i -e 's/Simple_Dungeon.nbt/simple_dungeon.nbt/g' *`

This is Linux related issue.

**Comment by MCPitman** [comment]
Fixed in v6

**Comment by DmitryRendov** [comment]
Guys, 7 files are left to fix
![image](https://user-images.githubusercontent.com/1677593/33955230-e147aecc-e04b-11e7-844a-b5e9471005c1.png)

fyi @MCPitman

**Comment by DmitryRendov** [comment]
can confirm, resolved in V6.1
